assets/js/util.js

```diff
@@ -242,8 +243,26 @@ function synchronizeTextareaHeights() {
     $('#translatedText').css('height', originalTextScrollHeight + 'px');
 }
 
-/*:: export {synchronizeTextareaHeights, modeEnabled, ajaxSend, ajaxComplete, filterLangList, onlyUnique,
-    SPACE_KEY_CODE, ENTER_KEY_CODE, HTTP_OK_CODE, HTTP_BAD_REQUEST_CODE, XHR_LOADING, XHR_DONE} */
+function apyCall(requestObject, endpoint) {
+    window[requestObject.requestName] = $.ajax({
```

Comment:
@sushain97 ! Um, 

> With the Object.assign method, a call-site could include extra parameters to $.ajax without modifying the apyCall function itself.

What would the `target` object hold in this case?  Also, why shouldn't I directly use `callApy(options, endpoint, requestName)` ? Should the `target` object be a JS object with `key`: `value` pairs corresponding to request names? 

( Not understanding how I have to use `Object.assign` function! ) A code snippet would be nice :D